scripts/db_migrator.py

```diff
@@ -47,7 +47,8 @@ def __init__(self, namespace, socket=None):
                      none-zero values.
               build: sequentially increase within a minor version domain.
         """
-        self.CURRENT_VERSION = 'version_4_0_3'
+        
+        self.CURRENT_VERSION = 'version_4_0_4'
```

Comment:
Looks like the current version on master is already 4_0_4. Please rebase w/ master branch and update to a newer version.

Reply:
Rebased to master and pushed the changes. Pls review.